src/main/java/cat/udl/eps/softarch/demo/repository/ShelterCertRepository.java

```diff
@@ -0,0 +1,9 @@
+package cat.udl.eps.softarch.demo.repository;
+
+import cat.udl.eps.softarch.demo.domain.ShelterCertificate;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+public interface ShelterCertRepository extends CrudRepository<ShelterCertificate, Long>, PagingAndSortingRepository<ShelterCertificate, Long> {
```

Comment:
Hi, rename the interface using the complete name, "ShelterCertificationRepository" I know that is a long name, but in some cases, it can avoid confusion. Thanks!!

Good Job!

Reply:
Resolved